shpc/tests/test_wrappers.py

```diff
@@ -0,0 +1,118 @@
+#!/usr/bin/python
+
+# Copyright (C) 2022 Vanessa Sochat.
+
+# This Source Code Form is subject to the terms of the
+# Mozilla Public License, v. 2.0. If a copy of the MPL was not distributed
+# with this file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import pytest
+import os
+
+import shpc.main.container as container
+import shpc.main.wrappers.base as wrappers_base
+
+from .helpers import init_client, here
+
+
+@pytest.mark.parametrize(
+    "with_custom_wrapper_template_path,include_container_dir",
+    [
+        (False, False),
+        (False, True),
+        (True, False),
+        (True, True),
+    ],
+)
+def test_get_template_paths(
+    tmp_path, with_custom_wrapper_template_path, include_container_dir
+):
+    """
+    Test the shpc.main.wrappers.base.WrapperScript.get_template_paths function
+    """
+    # Note: the module_sys and container_tech parameters are actually not used in the test
+    client = init_client(str(tmp_path), "tcl", "singularity")
+    # The file name is actually not used in the test, only its enclosing directory
+    testdata_dir = os.path.join(here, "testdata")
+    container_yaml = os.path.join(testdata_dir, "quay-container.yaml")
+    config = container.ContainerConfig(container_yaml)
+    ws = wrappers_base.WrapperScript(
+        "dummy.sh",  # Not used in this test
+        client.settings,
+        None,  # image
```

Comment:
nit: do these not have defaults of None? If we use kwargs for the config do we need this here?

Reply:
I fixed the `None` of the "container" parameter but I can't directly get rid of the `None` of the "image" parameter because it's a positional argument, not a keyword one. As a solution, I could pass the first two as keywords, or turn "image" to a keyword one

Reply:
Don’t worry about that - your change here should be good!

Reply:
If we were to expose this class externally we could maybe think harder about this, but since it’s an internal one I don’t think we need to go further than what we have for the test.